server/utils/middleware.js

```diff
@@ -29,6 +29,18 @@ export function ifNoUser401(req, res, next) {
   return res.status(401).end();
 }
 
+export function isNamed(req, res, next) {
+  if (req.user.name === '') {
+    return res.status(200).send(`We need your name so we can put it on your
```

Comment:
Feels like this string wants to be `dedent`ified too:
![](http://i.prntscr.com/b04bed7a486b43b5a08854a49e7ea51e.png)